test/k8sT/manifests/1.10/v1.0/cilium-ds-patch.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I'm wary of this... is there a reason why we can't use patches from the v1.0, v1.1, v1.2 repos themselves? Also, how will these be updated if the generated YAMLs are updated as well with new arguments, etc.?

Reply:
> is there a reason why we can't use patches from the v1.0, v1.1, v1.2 repos themselves?

Can you point me to where are they in the `v1.x` tags? ;-)

We can backport these files to those branches but the logic would proven to be difficult as we check if the file exists, if it doesn't we fall back to the one in `test/k8sT/manifests/`

>Also, how will these be updated if the generated YAMLs are updated as well with new arguments, etc.?

These files shouldn't be changed because the arguments don't change. You might not have realized but the files in v1.2 are different from the files in v1.1 because there are arguments in v1.2 that don't exist in v1.1 for example.

Reply:
I'm not too set one way or another on this, so it's fine.